config/skaffold/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+## This folder is strictly for local development work using skaffold.
```

Comment:
I figured it was easiest to create a new kustomize target for this to be used for dev stuff only. I'll try to include more parts of the system in this over time as the need arises - e.g. installing cert-manager and nfd as pre-reqs.